wbooks/src/app/screens/DetailBook/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const ADD_WISH = 'Add to whislist';
+export const CHECK = '✔';
+export const EMPTY = '';
```

Comment:
use an enum too

Reply:
I think is not necessary here

Reply:
I think you just need to set a single object with all of these values, but not an enum since i dont see the value of doing that.